VirtualList/tests/useEvent-specs.js

```diff
@@ -0,0 +1,204 @@
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
+
+import Item from '../../Item';
+import VirtualList from '../VirtualList';
+
+const focus = (elm) => fireEvent.focus(elm);
+
+const keyDownUp = (keyCode) => (elm) => {
+	fireEvent.keyDown(elm, {keyCode});
+	return fireEvent.keyUp(elm, {keyCode});
+};
+
+const leftKey = keyDownUp(37);
+const rightKey = keyDownUp(39);
+const upKey = keyDownUp(38);
+const downKey = keyDownUp(40);
+const pageDownKey = keyDownUp(34);
```

Comment:
How about renaming these to `pressSomeKey` ?

Reply:
I fixed it